core/tests/puppeteer-acceptance-tests/voiceover-tests/play-uploaded-recording.js

```diff
@@ -0,0 +1,34 @@
+const acceptanceTests = require("../utility-functions/puppeteer_utils.js");
```

Comment:
@Shivkant-Chauhan @ashish-patwal See if you can make these more readable as well.

Reply:
done